addons/im_livechat/models/mail_channel.py

```diff
@@ -36,6 +36,13 @@ class MailChannel(models.Model):
     channel_type = fields.Selection(selection_add=[('livechat', 'Livechat Conversation')])
     livechat_channel_id = fields.Many2one('im_livechat.channel', 'Channel')
 
+    @api.multi
+    def _compute_is_chat(self):
+        super(MailChannel, self)._compute_is_chat()
+        for record in self:
+            if record.channel_type == 'livechat':
```

Comment:
shouldn't be `if record.channel_type in ['livechat', 'chat']`?

Reply:
in module `mail`, it sets `is_chat` when this is a channel of type `chat`. The module `im_livechat` extends this behaviour by including channels of type `livechat`.

We do not want to specify `in ['livechat', 'chat']` in `im_livechat`, because if another module (let's say, `im_support`) defines another kind of channel that should be considered as a chat, we have high dependency among modules.

Reply:
yeah, I see now, thanks